juntagrico/templates/job.html

```diff
@@ -15,13 +15,12 @@ <h3>
 {% endblock %}
 {% block content %}
     {% vocabulary "assignment_pl" as v_assignment_pl %}
-    <div class="row">
+    <div class="row pb-4">
         <div class="col-md-3">
             {% trans "Zeitpunkt" %}:
         </div>
         <div class="col-md-5">
-            {{ job.time |date:"l, d.m.Y, H:i" }} - {{ job.end_time|date:"H:i" }}
-            <br/>
+            <div class="pb-4">{{ job.time |date:"l, d.m.Y, H:i" }} - {{ job.end_time|date:"H:i" }}</div>
```

Comment:
I think here the br was legit, as it was not used to add spacing but to break this line.

Reply:
reverted